internal/host/agent_server_grpc/proto/service.proto

```diff
@@ -135,14 +137,14 @@ message RemoveRequest {
 message RunRequest {
   string path = 1;
   repeated string args = 2;
-  repeated string env = 3;
+  repeated string env_vars = 3;
```

Comment:
protolint wans to do `envs` (which LGTM), but it gets a bit confusing with `Env` on `Cmd`... I decided for `env_vars` instead.

Reply:
Cool